CITATION.cff

```diff
@@ -9,6 +9,9 @@ authors:
   given-names: Eleanor
   orcid: https://orcid.org/0000-0001-8773-7838
   affiliation: Institute of Oceanography, University of Hamburg, Bundesstraße 53, 20146 Hamburg, Germany
+- family-names: Monforte
+  given-names: Chiara
```

Comment:
do you have an orcid? If not, sign up! It's very useful for identifying researcher contributions https://orcid.org/

Reply:
I do not hahah I will create one and add it. Feel like an imposter though haha